tests/InputFiles/ExportCoordinates/Input1D.yaml

```diff
@@ -6,13 +6,29 @@
 # ExpectedOutput:
 #   ExportCoordinates1DVolume0.h5
 
+Evolution:
+  InitialTime: 0.0
+  InitialTimeStep: 0.01
+  TimeStepper:
+    AdamsBashforthN:
+      Order: 1
+
 DomainCreator:
-  Interval:
-    LowerBound: [0]
-    UpperBound: [1]
-    IsPeriodicIn: [false]
-    InitialRefinement: [1]
-    InitialGridPoints: [3]
+    Interval:
+      LowerBound: [-0.5]
+      UpperBound: [0.5]
+      IsPeriodicIn: [false]
+      InitialRefinement: [2]
+      InitialGridPoints: [5]
+      TimeDependence:
+        UniformTranslation:
+          InitialTime: 0.0
+          Velocity: [0.5]
+          FunctionOfTimeNames: ["TranslationX"]
+
+EventsAndTriggers:
+  ? PastTime: 1.0
+  : - Completion
```

Comment:
What do I need to change in this input file to export a static domain (once)? Perhaps make one of the input files an example for that and/or add comments to the files. I think a person looking at these files shouldn't need to figure that out by themselves. If I set `PastTime: 0.` I currently get two observations. If I set it to -1 I get only one, but that's a non-obvious thing to do.

I suggest this:
- Move the `Evolution` section to the very bottom, after `EventsAndTriggers`.
- Add a comment here like "Set `PastTime: 0.` (or whatever it should be) to export a domain with no time dependence"

Reply:
This is caused by strict vs. non-strict inequalities.  (`PastTime: 0` checks `t > 0`, not `t >= 0`.)  I suggest `Always`.